src/components/configurator/Features.js

```diff
@@ -259,6 +263,7 @@ export default class Features extends React.Component {
             onMouseLeave={onMouseLeave}
             selectedBuildTool={selectedBuildTool}
             key={group}
+            expandByDefault={_.size(groupedFeatures) === 1}
```

Comment:
Correct me if I'm wrong, but now that we have React, we don't need passing in `expandByDefault` and the implementation in `FeatureGruop`?

Reply:
Yes, we can remove it. But having it won't hurt, in case we are going to add another tab in future this logic may be useful ?

Reply:
That's a valid point, however, in my experience all line of code adds up to creating complexity so I'm very careful not to have more code than necessary. 

If you create a new commit reverting these lines, we'll have the code in git history and we can easily get them back if needed.